### DISCUSSION TITLE
奇忽局02号理事会：奇忽局可持续发展&激励机制理事会

### DISCUSSION THREAD

**Discussion by Lupingyue** [qihuju/qihuju.github.io#18] [General]
定义和规则有待补充……

以下用于抛砖引玉，by @askender, 欢迎参与编辑，稍后会在wiki里不断完善。
定义（草稿）： 三方面的含义：如同在奇忽局共识1.0中说的一样，开源和创作共用CC许可是可持续的基础保障；长期存储方面偏技术可以加一些ipfs的玩法；激励机制是保障贡献者得到回报的最佳方法，目前不采用公司形式，而是借用token激励的方式，后面会持续实验。

规则&玩法（草稿）（edited by @askender ）：

- 添加评论即可加入或者退出
- 每周在ch简单交流一次，采用罗伯特议事原则
  1. [Wikipedia - 罗伯特议事原则](https://zh.wikipedia.org/zh/%E7%BD%97%E4%BC%AF%E7%89%B9%E8%AE%AE%E4%BA%8B%E8%A7%84%E5%88%99)
  2. [简书 - 《罗伯特议事规则》的12条基本原则（极简版）](https://www.jianshu.com/p/bd33834dd44e)
  3. [知乎 - 罗伯特议事规则：用12原则6步法，开好一个会](https://zhuanlan.zhihu.com/p/29639932)
- 暂定大致分  开源项目规划小组、玩法创新小组、激励分发小组
- 有其他想法可以编辑规则和玩法（不要直接删除文字）
- 相关具体计划产出稍后使用git-repo来保存和更新。
- 欢迎任何人参与讨论和开发

**Comment by askender** [comment]
相关链接： https://github.com/qihuju/qihuju.github.io/discussions/5
定义和规则有些抽象，先抛砖引玉一下：
大致是关于： 提供一个孵化机制，激励有兴趣的志同道合的人抽出时间做去中心化的体验好的产品，并帮助获得后续产品持续进化的可能性。
找到合适的人（他们自己来描述项目），提供合适的环境和资源（合作者开发者资源、开发生态资源、token激励等）
之后有空再去ch沟通一下

**Comment by Lupingyue** [comment]
> 相关链接： https://github.com/qihuju/qihuju.github.io/discussions/5
> 
> 定义和规则有些抽象，先抛砖引玉一下：
> 
> 大致是关于： 提供一个孵化机制，激励有兴趣的志同道合的人抽出时间做去中心化的体验好的产品，并帮助获得后续产品持续进化的可能性。
> 
> 找到合适的人（他们自己来描述项目），提供合适的环境和资源（合作者开发者资源、开发生态资源、token激励等）
> 
> 之后有空再去ch沟通一下
> 
> 

这段初始定义调性可以的。不断充实。

**Comment by Tz-H** [comment]
【致富信息】

Clubhouse 最新更新已经灰度开放打赏功能，详情：

- [Clubhouse launches payments so creators can make money
](https://techcrunch.com/2021/04/05/clubhouse-payments/)

声音实践理事会 #34 或许可以成为这个功能的践行者，进行打赏功能的相关测试

**Reply by Lupingyue** [reply]
这个事挺重要

**Reply by Tz-H** [reply]
于4月15日 在 Clubhouse 房间中成功进行了一次打赏操作。

- 打赏对象：`Zheng Yaohua @shanghaipie`
- 打赏金额：USD 5.00
- 实际支付：USD 5.46
- 汇率转换金额：GBP 3.97
- 支付方式：信用卡（万事达/Mastercard）
- 信用卡标注接收方：Clubhouse

**Comment by Tz-H** [comment]
【致富信息】

Line 上可以创作并售卖表情包，详情：

[Line Creators Market](https://creator.line.me/zh-hans/guideline/sticker/)

可以与话题 #3 以及 #11 相关联。

表情包的市场拓展方向还包括了[微信表情开放平台](https://sticker.weixin.qq.com/cgi-bin/mmemoticon-bin/loginpage?t=login/index)以及[Telegram Stickers](https://telegram.org/blog/discover-stickers-and-more)。

**Reply by Lupingyue** [reply]
看来要激活一下line，我之前注册了，但是一直没有怎么用

**Reply by Sleeper2060** [reply]
制作表情包的局内有感兴趣的艺术家画家参与了吗？

**Reply by Lupingyue** [reply]
局内的艺术家好像没有擅长开发表情包的

**Reply by Sleeper2060** [reply]
下周我试着在群里问问看。

**Reply by Lupingyue** [reply]
> 下周我试着在群里问问看。

👌

**Comment by askender** [comment]
## 激励机制实验1：
领空投玩法和流程：
- 看介绍：使用的第一个激励token是likecoin:  https://like.co/    下一版会更新的内容 https://iscn.io/
- 下载cosmos生态下的钱包扩展： https://chrome.google.com/webstore/detail/keplr/dmkamcknogkgcdfhhbddcghachkejeap  并生成钱包地址，注意记下助记词，这是唯一能恢复钱包的方法。
- 访问去中心化钱包（这两个链接随意选一个即可）获得自己的likecoin地址：  https://likecoin.on.fleek.co/     https://likecoin-chain-sheungwan.netlify.app/   可以回复此讨论来领取空投。  
- 领取like最好完成以下步骤：  1.  加入 奇忽局discord:  https://discord.gg/tKGpkTKnar  初期默认10个；2 如果有社区贡献，比如参与clubhouse, github, discord讨论等会更多； 如果是Arctic Code Vault Contributor也有额外奖励；先来的可能也有额外激励； 如果也加入了liker社区 https://discord.gg/W4DQ6peZZZ  也有额外激励。   空投发起人有最终解释权。后面会更新补充玩法。

信息补充：
- 如果以上流程遇到问题，可以在 奇忽局discord: https://discord.gg/tKGpkTKnar  #airdrop 频道咨询。也可以在这里直接问。

**Reply by Lupingyue** [reply]
感觉有点意思

**Reply by Lupingyue** [reply]
刚才试着操作了一下，感觉还是有点复杂，待会你来培训一下

**Reply by Tz-H** [reply]
@askender 

Keplr 钱包已注册

cosmos 钱包地址：cosmos1aq9mxtlvwh50g7d9ra9z6jhkv04rj636mg3wpq

**Reply by askender** [reply]
@Tz-H    已经空投100like, 这里也能看到： https://likecoin.bigdipper.live/account/cosmos1aq9mxtlvwh50g7d9ra9z6jhkv04rj636mg3wpq

**Reply by Tz-H** [reply]
> @Tz-H 已经空投100like, 这里也能看到： https://likecoin.bigdipper.live/account/cosmos1aq9mxtlvwh50g7d9ra9z6jhkv04rj636mg3wpq

确认地址已收到 Likecoin 100 units.

截图如下：
![Screenshot_18_04_2021__16_18](https://user-images.githubusercontent.com/1928161/115150771-c4dcab00-a061-11eb-8bbe-cb2ad08cad0c.jpg)

还需要更多的玩法介绍和推荐以及使用方式来使用这些 Likecoin。 @askender

**Reply by MingxuanXie** [reply]
@askender 钱包地址：cosmos106vg78c95zevu5k6uz823emt04mn6qnu33hnvp
说起来我竟然是Arctic Code Vault Contributor😂，然而被选入冰封起来的只是我瞎搞各种实验的个人网站，，全是黑历史。。。

**Comment by Lupingyue** [comment]
![屏幕快照 2021-04-15 下午8 48 41](https://user-images.githubusercontent.com/25149468/114872567-0d5f5300-9e2d-11eb-9c75-bf1bdafe6f2c.png)

我的第一笔like coin，截图纪念一下😄

**Reply by askender** [reply]
当时空投账号里没存like， 现在已经重新空投100like,   也可以在区块浏览器上看到：  https://likecoin.bigdipper.live/account/cosmos10gwn9kclct7wgtv97plk8p4ccznzvfvdzw68a5

**Reply by Sleeper2060** [reply]
@askender
Keplr 钱包钱包地址：cosmos16wx7tmusufevf8d4mgkf2ckg0ej0xtd9zz3j6e
